frontend/src/ui/metadata.tsx

```diff
@@ -21,12 +21,16 @@ export const InputContainer: React.FC<{ children: ReactNode }> = ({ children })
     <div css={{ margin: "16px 0 " }}>{children}</div>
 );
 
-type DescriptionProps = {
+type SmallDescriptionProps = {
     text: string | null;
     lines?: number;
 };
 
-export const Description: React.FC<DescriptionProps> = ({ text, lines = 2 }) => {
+/**
+ * An event's or series' description in at most `lines` lines. Not rendered at
```

Comment:
I would qualify the term "rendered" here somehow. Because in a React sense, that text is certainly being rendered.

Or just replace the word with "transformed" or "interpreted" or sth.